### DISCUSSION TITLE
A command to stop play_sound

### DISCUSSION THREAD

**Discussion by Skeletorjus** [rand256/valetudo#378] [Ideas]
It'd be useful to have a command, for example "stop_sound", that would stop the currently playing audio.
Use case could be to stop the audio in order to start a new one (maybe a notification of sorts) or to simply stop whatever long audio track you started by accident.
The only way I've found to stop the audio manually is by SSH'ing into the vacuum and run a pkill command, but that is quite cumbersome :)

**Comment by rand256** [comment]
This is implemented in 0.10.4. To stop a playback, call the "play_sound" command with a "file" (or "location" - doesn't matter) argument set to a string "none". Thanks for suggestion.
